### ISSUE TITLE
The copy button does not copy but loads the next string

### ISSUE THREAD

**Issue by nblock** [nijel/weblate#1103]
### Steps to reproduce
1. Open a project for translation
2. Click the copy button to copy from source to translation

### Actual behaviour
When the copy button is clicked it loads the next translation.

### Expected behaviour
Copy the source into translation and tick the "Needs review" flag

### Server configuration
```
 * Weblate 2.6
 * Python 2.7.9
 * Django 1.8
 * six 1.9.0
 * python-social-auth 0.2.7
 * Translate Toolkit 1.14.0-rc1
 * Whoosh 2.7.0
 * Git 2.1.4
 * Pillow (PIL) 1.1.7
 * dateutil 2.4.2
 * lxml 3.4.4
 * django-crispy-forms 1.4.0
 * compressor 1.5
 * djangorestframework 3.3.3
 * pytz 2015.2
 * pyuca N/A
 * pyLibravatar N/A
 * Mercurial 3.3.3
 * Database backends: django.db.backends.postgresql_psycopg2
```

**Comment by nijel** [comment]
This could probably happen in case the file was changed meanwhile, are you able to reproduce it?

Anyway this would be eventually fixed with #1038.

**Comment by nblock** [comment]
Yes, I can reproduce it with any project.

**Comment by nijel** [comment]
Can you check with developer tools in your browser how does it request the translation from the server?

**Comment by nblock** [comment]
I tried with a new chromium session using (```--temp-profile```)

This is what the developer tools (network) lists when I click on the 'copy' button
http://i.imgur.com/NN0keCy.png

**Comment by nijel** [comment]
Okay, so it just submits the form and really doesn't request the translation...

**Comment by nijel** [comment]
This should be fixed with fix for #1038.
